src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/Datasets/SetAzureDataFactoryDatasetCommand.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Commands.DataFactoryV2
 {
     [Cmdlet(VerbsCommon.Set, Constants.Dataset, DefaultParameterSetName = ParameterSetNames.ByFactoryName,
         SupportsShouldProcess = true), OutputType(typeof(PSDataset))]
-    [Alias(VerbsCommon.New + "-" + Constants.DataFactory)]
+    [Alias(VerbsCommon.New + "-" + Constants.Dataset)]
```

Comment:
Please add back the old alias - this would be considered a breaking change.  If you want to deprecate the old one, please do so, and we'll remove in November